src/uart.rs

```diff
@@ -409,7 +512,19 @@ pub mod config {
                 flow_control: FlowControl::None,
                 flow_control_rts_threshold: 122,
                 source_clock: SourceClock::default(),
-                intr_flags: EnumSet::<IntrFlags>::empty(),
+                intr_alloc_flags: EnumSet::<IntrFlags>::empty(),
```

Comment:
Note: This is a breaking change.

Reply:
Might not be necessary with the rename of the "other" interrupt flags to `event_flags` or suchlike.